### PR TITLE
Aerogear 3549

### PR DESCRIPTION
**Pull request by josemigallas**

### CODE REVIEW

File: ui/components/data-sources/DataSourcesList.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import { Query } from "react-apollo";
+import { ListView } from "patternfly-react";
+
+import GetDataSources from "../../graphql/GetDataSources.graphql";
```
Comment (pb82) [inline ui/components/data-sources/DataSourcesList.js]: This is nice

**Comment by josemigallas** [comment]
Rebased

File: ui/components/data-sources/DataSourcesListItem.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+        <ListView.Item
+            className="ds-list-item"
+            heading={type}
+            description="---"
+            leftContent={(
+                <span className="list-item-name">{name}</span>
+            )}
+            actions={(
+                <div>
+                    <DropdownKebab id="data-source-list-item-dropdown" pullRight>
```
Comment (pb82) [inline ui/components/data-sources/DataSourcesListItem.js]: You can already add the real options: `Edit Data Source` and `Delete Data Source`

File: ui/index.js [hunk 3]
```diff
@@ -1,47 +1,15 @@
...
-                            <Nav bsClass="nav nav-tabs">
-                                <NavItem disabled={false}>
-                                    <div>Data Sources</div>
-                                </NavItem>
-                            </Nav>
-                            <TabContent>
-                                <TabPane>
-                                    <DataSourcesView />
-                                </TabPane>
+const client = new ApolloClient();
```
Comment (pb82) [inline ui/index.js]: Is `/graphql` the default uri? No other config required?

**Comment by cfoskin** [comment]
@josemigallas Looks good

File: ui/index.js (hunk 3 above)
Comment (josemigallas) [reply ui/index.js]: That is correct https://www.apollographql.com/docs/react/essentials/get-started.html#creating-client

**Pull request merged**